backend/src/main/java/com/woowacourse/naepyeon/repository/message/MessageRepositoryImpl.java

```diff
@@ -23,10 +33,50 @@ public List<Message> findAllByRollingpaperId(final Long rollingpaperId) {
                 .fetch();
     }
 
+    @Override
+    public Page<WrittenMessageResponseDto> findAllByAuthorId(final Long authorId, final Pageable pageRequest) {
+        final List<WrittenMessageResponseDto> content = queryFactory
+                .select(makeProjections())
+                .distinct()
+                .from(message)
+                .join(message.rollingpaper, rollingpaper)
+                .join(message.rollingpaper.team, team)
+                .join(message.rollingpaper.teamParticipation, teamParticipation)
+                .where(isAuthorIdEq(authorId))
+                .offset(pageRequest.getOffset())
+                .limit(pageRequest.getPageSize())
+                .fetch();
+
+        final JPAQuery<Long> countQuery = queryFactory
+                .select(message.countDistinct())
+                .from(message)
+                .where(isAuthorIdEq(authorId));
+
+        return PageableExecutionUtils.getPage(content, pageRequest, countQuery::fetchOne);
+    }
+
+    private ConstructorExpression<WrittenMessageResponseDto> makeProjections() {
+        return Projections.constructor(WrittenMessageResponseDto.class,
+                message.id,
+                message.rollingpaper.id,
+                message.rollingpaper.title,
+                message.rollingpaper.team.id,
+                message.rollingpaper.team.name,
```

Comment:
```suggestion
             team.name,
```
위의 코멘트와 연결되는 부분입니다. 이렇게 위와 같이 변경하면 좋을 것 같아요!
구조가 변경된다면 querydsl 구현하실때 참고하면 좋을 것 같아요